wdl-format/src/v1.rs

```diff
@@ -0,0 +1,414 @@
+//! A module for formatting WDL v1 elements.
+
+use wdl_ast::v1::Decl;
+use wdl_ast::v1::DocumentItem;
+use wdl_ast::v1::Expr;
+use wdl_ast::v1::HintsItem;
+use wdl_ast::v1::HintsSection;
+use wdl_ast::v1::InputSection;
+use wdl_ast::v1::LiteralBoolean;
+use wdl_ast::v1::LiteralFloat;
+use wdl_ast::v1::LiteralInteger;
+use wdl_ast::v1::LiteralString;
+use wdl_ast::v1::OutputSection;
+use wdl_ast::v1::StringPart;
+use wdl_ast::v1::StructDefinition;
+use wdl_ast::v1::Type;
+use wdl_ast::AstNode;
+use wdl_ast::AstToken;
+use wdl_ast::SyntaxElement;
+use wdl_ast::SyntaxKind;
+
+use super::comments::format_inline_comment;
+use super::comments::format_preceding_comments;
+use super::first_child_of_kind;
+use super::state::SPACE;
+use super::Formattable;
+use super::State;
+use super::NEWLINE;
+use super::STRING_TERMINATOR;
+
+impl Formattable for LiteralString {
+    fn format<T: std::fmt::Write>(&self, writer: &mut T, _state: &mut State) -> std::fmt::Result {
+        write!(writer, "{}", STRING_TERMINATOR)?;
+        for part in self.parts() {
+            match part {
+                StringPart::Text(text) => {
+                    write!(writer, "{}", text.as_str())?;
+                }
+                StringPart::Placeholder(placeholder) => {
+                    write!(writer, "{}", placeholder.syntax())?;
```

Comment:
Does this write the placeholder verbatim? Should we be formatting it's expression?

Reply:
Answered above.